client/host_test.go

```diff
@@ -54,3 +57,56 @@ func TestGetHostByName(t *testing.T) {
 	}
 
 }
+
+func TestGetSortedHosts(t *testing.T) {
+	c, err := NewClient(GetConfigFromEnv())
+	if err != nil {
+		t.Fatalf("failed to create client with error: %v", err)
+	}
+
+	poolName := accTestHost.Pool
+	hosts, err := c.GetSortedHosts(Host{Pool: poolName}, "id", "asc")
+	if err != nil {
+		t.Fatalf("failed to get host with error: %v", err)
+	}
+	if len(hosts) == 0 {
+		t.Errorf("failed to find any host for pool `%s`.", poolName)
+	}
+	if len(hosts) > 1 {
```

Comment:
It would be great to exercise this deterministically rather than hoping at runtime we have more than 1 host

Reply:
The acceptance tests will fail if there aren't more than 1 host so I'm ok with leaving this as is.